doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,133 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+Lots of GitHub issues and pull requests are being created daily.
+To help us manage them we have “labels”:
+to classify them by area, type, priority, or to mark them in need of attention.
+
+All GitHub issues or pull requests must be appropriately labeled.
+Issues and PRs often have multiple labels assigned,
+to help classify them in the different available categories.
+When reviewing a PR, if it has missing or incorrect labels, maintainers shall
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, reduces the changes of duplicates, etc.
+
+These are the labels we currently have, grouped by type:
+
+Area:
+*****
+
+=============  ===============================================================
+Labels         ``Area:*``
+Applicable to  PRs  and issues
+Description    Indicates subsystems (e.g., Kernel, I2C, Memory Management),
+               project functions (e.g., Debugging, Documentation, Process),
+               or other categories (e.g., Coding Style, MISRA-C)  affected by
+               the bug or pull request.
+=============  ===============================================================
+
+An area maintainer should be able to filter by an area label and
+find all issues and PRs which relate to that area.
+
+Platform
+*********
+
+=============  ===============================================================
+Labels         ``Platform:*``
+Applicable to  PRs  and issues
+Description    An issue or PR which affects only a particular platform
+=============  ===============================================================
+
+To be discussed in a meeting
+****************************
+
+=============  ===============================================================
+Labels         ``API``, ``dev-review``, ``TSC``
+Applicable to  PRs  and issues
+Description    The issue is to be discussed in the following API/dev-review/TSC
+               meeting if time permits.
+=============  ===============================================================
+
+Minimum PR review time
+**********************
+
+=============  ===============================================================
+Labels         ``Hot Fix``, ``Trivial PR``, ``Maintainer``,
```

Comment:
isn't the labe "Trivial" and not "Trivial PR"?

Reply:
The plan was to rename Trivial => "Trivial PR" (To avoid developers assigning it to trivial bugs.

Reply:
Does it matter if we reuse the label for bugs and PRs?

Reply:
If there is any purpose of using a "Trivial" label in a bug then we could use it. Today, I don't see any need for such.
But having too many labels with no clear purpose will add to the confusion (which we are trying to reduce with this PR)